test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -32,9 +32,7 @@ const (
 )
 
 // These tests don't seem to be running properly in parallel: issue: #20338.
-//
-
-var _ = Describe("Horizontal pod autoscaling (scale resource: CPU)", func() {
+var _ = KubeDescribe("Horizontal pod autoscaling (scale resource: CPU) [Serial] [Slow]", func() {
```

Comment:
The old describe included these, but they're not needed anymore as there is now the light test, ill remove them